### ISSUE TITLE
About your new model

### ISSUE THREAD

**Issue by Drsea123** [HsuTzuJen/Face_Recognition_Practice_with_TF#1]
Could you tell me about your model's learning rate and How much steps you can arrival the 99+ accuracy for your model? please! @HsuTzuJen

**Comment by HsuTzuJen** [comment]
Please check the Repository " Face_Recognition_Practice_with_TF"
I have updated.

**Comment by Drsea123** [comment]
感谢你的模型，当我从train.idx中分离出前10000类进行训练时，在进行40000次训练后，inference loss中总在16左右徘徊，不会再下降，我想请教您一下这是为什么？

**Comment by Drsea123** [comment]
@HsuTzuJen

**Comment by HsuTzuJen** [comment]
The value of inference loss is for reference only. It does not mean whether the training is absolutely good or bad.
The inference loss is bigger than original since we have modified it by minus a margin of the dot product between the weight of y^ and feature.  Actually, I also add 1 to the dot product between the weights of not y^ to prevent not convergence in the early training steps, so the inference loss is bigger than the original cosine face loss.

**Comment by Drsea123** [comment]
Please forgive me for being a newbie. If I want to enter two pictures into the model to test their similarity and get the network output, how can I judge their similarity through the output of the network? How do I edit this script?

**Comment by Drsea123** [comment]
Excuse me, When  I have been training your model, the training accuracy always was zero?why? @HsuTzuJen

**Comment by HsuTzuJen** [comment]
1. normalize the features and compare them with Euclidean distance or cosine similarity.
2. The loss function is modified with margins, you may have to read the paper large margin cosine loss.

**Comment by Drsea123** [comment]
You don't use arcface loss ? why?

**Comment by Drsea123** [comment]
@HsuTzuJen

**Comment by HsuTzuJen** [comment]
It is not always converged with this network.
You can try it. 
It is in the face_losses.py.
Remember to modify the code without using dynamic margin.

**Comment by Joll123** [comment]
@HsuTzuJen 我想把meta和ckpt转为pb模型用于人脸验证，模型的输出节点是什么。
![image](https://user-images.githubusercontent.com/43200569/48185873-5617a580-e371-11e8-8c3c-1ec06a7de4c9.png)
谢谢
